oio/cli/common/clientmanager.py

```diff
@@ -68,7 +72,8 @@ def flatns_manager(self):
         options = self.nsinfo['options']
         bitlength, offset, size = None, 0, None
         try:
-            bitlength = int(options['flat_bitlength'])
+            bitlength = int(self._flatns_config.get('flat_bitlength',
+                            options['flat_bitlength']))
```

Comment:
In order to harmonize parameters, I would call it `flat_bits` (as in `sds.conf` files).